src/api/files.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const argCommand = require('../cmd-helpers').argCommand
+
+module.exports = send => {
+  return {
+    cp: argCommand(send, 'files/cp'),
+    ls: argCommand(send, 'files/ls'),
+    mkdir: argCommand(send, 'files/mkdir'),
+    stat: argCommand(send, 'files/stat'),
+    rm: function (path, opts, cb) {
```

Comment:
oh, for sure! I was just checking the commit history and I had that before, dunno why I removed it meanwhile, sorry and thank you for catching that :)